back/api/admin.py

```diff
@@ -80,22 +80,13 @@ class PricingAdmin(CustomModelAdmin):
 
 
 class UserAdmin(BaseUserAdmin):
-    """Overrides BaseUserAdmin in order to hide some defined but not used fields"""
-    fieldsets = (
-        (None, {'fields': ('username', 'password')}),
-        (_('Personal info'), {'fields': ('first_name', 'last_name', 'email')}),
-        (_('Permissions'), {
-            'fields': ('is_active', 'is_staff', 'is_superuser'),
-        }),
-        (_('Important dates'), {'fields': ('last_login', 'date_joined')}),
-    )
-    filter_horizontal = []
+    """Overrides BaseUserAdmin"""
+
     inlines = [IdentityInline]
 
 
-admin.site.unregister(User)
-admin.site.unregister(Group)
-admin.site.register(User, UserAdmin)
+admin.site.unregister(UserModel)
+admin.site.register(UserModel, UserAdmin)
```

Comment:
I do not get these two lines... You unregistrer and register `UserModel`. Why is it first necessary to unregistrer `UserModel`?